draft-tigress-requirements.md

```diff
@@ -128,6 +128,25 @@ General terms:
 
 - Bob booked a room at a hotel for the weekend, but will be arriving late at night. Alice, his partner, comes to the hotel first, so Bob wants to share his key to the room with Alice. Bob and Alice are using two different mobile phones with different operating systems. In order for Bob to share his key to the hotel to Alice for a weekend, he must transfer some data to her device. The data structure shared between the two participants is proprietary to the given hotel chain (or Provisioning Partner). This data transfer is a one-time, unidirectional from Bob’s device to Alice’s. Once Alice receives this data, she can provision a new key to her device, making a call to Provisioning Partner to receive new credential information.
 
+# Relationships
+
+
+```mermaid
+sequenceDiagram
+    actor S as Sender
+    participant I as Intermediary
+    actor R as Receiver
+    S ->> I : upload credential data
+    break Generic messaging channel
+      S ->> R : send invite
+    end
+    Loop Provision credential
+      R ->> I : request credential data
+      I ->> R : deliver credetnai data
```

Comment:
```suggestion
      R ->> I : request credential data
      I ->> S : request credential data
      S ->> R : deliver credential data
      I ->> R : deliver credetnai data
```
Should we also include going to the sender to get the credential data?

Reply:
I'm confused with  `S ->> R : deliver credential data`, this looks to bypass the intermediary.  It would also mean the devices have to be online at the same time

Are you suggesting to illustrate the stateful flow?

Reply:
I think like 146 should be:
S ->> I : deliver credential data
The rest looks correct

Reply:
> I think like 146 should be (i took the liberty of updating Alex's suggested change):
> S ->> I : deliver credential data
> The rest looks correct

Ah yes, my bad. Thanks for updating!

Reply:
Merging